spec/lib/debt_management_center/financial_status_report_service_spec.rb

```diff
@@ -291,6 +295,7 @@ def mock_pdf_fill
       })
       allow_any_instance_of(DebtManagementCenter::VBS::Request).to receive(:post).and_return(response)
       mock_sharepoint_upload
+      allow(User).to receive(:find).with(user.uuid).and_return(user)
```

Comment:
instead of mocking the `find` for the `User` model here, just do a `let!` for the places that you `let` the necessary `User instance:

`let!(:user) { build(:user, :loa3) }`

Reply:
Doing this causes the spec to fail.